discovery/seata-discovery-raft/src/main/java/io/seata/discovery/registry/raft/RaftRegistryServiceImpl.java

```diff
@@ -470,6 +460,11 @@ public List<InetSocketAddress> lookup(String key) throws Exception {
                 if (CollectionUtils.isEmpty(list)) {
                     return null;
                 }
+                try {
+                    refreshToken(addresses[0]);
```

Comment:
```suggestion
                    refreshToken(queryHttpAddress(clusterName, key));
```
Why not get the address via `queryHttpAddress` here?